app/views/items/index.html.erb

```diff
@@ -17,18 +17,21 @@
   <tbody>
     <% @items.each do |item| %>
       <tr>
-        <td><%= item.icsid %></td>
-        <td><%= link_to item.model + " " + item.name, item_path(item) %></td>
-      	<td><%= item.location.name if item.location %></td>
+        <td><%= link_to item.name, item_path(item) %></td>
+        <td><%= item.model %></td>
+      	<td><%= item.location %></td>
       	<td><%= item.category %></td>
       	<td><%= item.serial1 %></td>
       	<td><%= item.serial2 %></td>
         <td><%= item.serial3 %></td>
-        <td><%= item.person.fullname unless item.person_id.blank? %></td>
-      	<td><%= item.status %></td>
+        <td><%= number_to_currency item.recent_costs %></td>
+      	<td>
+          <span class="label <%= 'label-warning' if item.status == 'Out of Service' %><%= 'label-success' if item.status == 'Available' %>">
```

Comment:
Could you shorten this line?  One approach would be to move some of this logic into the controller.